package.json

```diff
@@ -23,7 +23,7 @@
     "scripts/*"
   ],
   "scripts": {
-    "dev": "lerna run dev --parallel --ignore demo-*",
+    "dev": "lerna run dev --parallel",
```

Comment:
I'm ok with this. @cjpillsbury what was the reason again we ignored the examples?